### PR TITLE
Link to new contributing guide

### PR DESCRIPTION
**Pull request by gigamesh**
Link to new contributing guide

### CODE REVIEW

**Review by snario — APPROVED** [review]
We should remove this file in another couple of weeks, and just mention it in the README

**Comment by gigamesh** [comment]
> We should remove this file in another couple of weeks, and just mention it in the README

I actually just added the list of packages that only require a single reviewer. I'm thinking it makese sense for the monorepo & frontends repo to have their own contributing guides that sort of extend from the general guide. Thoughts?

**Comment by snario** [comment]
In general it makes sense to mention those things, yup!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,303 +1,14 @@
...
 
-🚨 Before making any non-trivial change via Pull Request, please first open an issue 
-describing the change to solicit feedback and guidance. This will increase the 
-likelihood of your Pull Request getting merged.
+**Please read our [general contributing guide](https://github.com/ethereum-optimism/.github/blob/master/CONTRIBUTING.md) before continuing**
 
-## Contributing in Issues
 
-For any issue, there are fundamentally three ways an individual can contribute:
+These packages only require 1 reviewer (all other packages require 2)
```
Comment (maurelian) [inline CONTRIBUTING.md:6]:
```suggestion
## PR Review Requirements

These packages only require 1 reviewer (all other packages require 2 reviewers, unless the changes do not affect production or test code). 
```

That's the intent of the guidelines, does that phrasing make sense?

Comment (gigamesh) [reply CONTRIBUTING.md]: added!

**Comment by platocrat** [comment]
updating this branch to up-to-date with `develop`

**Review by maurelian — APPROVED** [review]

**Pull request merged**